hudi-flink-datasource/hudi-flink/src/main/java/org/apache/hudi/sink/partitioner/profile/WriteProfiles.java

```diff
@@ -28,21 +28,23 @@
 import org.apache.hudi.common.util.Option;
 import org.apache.hudi.config.HoodieWriteConfig;
 import org.apache.hudi.exception.HoodieException;
-import org.apache.hudi.util.StreamerUtil;
 
 import org.apache.flink.core.fs.Path;
 import org.apache.hadoop.conf.Configuration;
 import org.apache.hadoop.fs.FileStatus;
 import org.apache.hadoop.fs.FileSystem;
+import org.apache.hudi.util.StreamerUtil;
 import org.slf4j.Logger;
```

Comment:
Need to fix the import sequence, take a reference of: https://github.com/apache/hudi/blob/d3ed4556c8c5cf4c3380ac573903c92abcffbb1d/style/checkstyle.xml#L291

Reply:
done